web/src/screens/LandingScreen/LandingScreen.pcss

```diff
@@ -227,10 +236,11 @@
   }
 
   &__register {
-    background-image: url(assets/images/landing/TradingUIFullBG.png);
-    background-position: right center;
-    background-repeat: no-repeat;
-    background-size: 80%;
+    background: url(assets/images/landing/TradingRegister.svg) no-repeat center center;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
```

Comment:
actually pcss generates this values after compilation, so you can remove that

Reply:
good to know this, thanks
will remove it now